readme.md

```diff
@@ -73,6 +77,19 @@ $ git init project-repo
 $ git add .
 $ git merge juan/feature
 ```
+> Solution
+```
+$ cd project-repo
+$ touch README.md
+$ git init project-repo
+$ git add .
+$ git commit -m "initial commit"
+$ git remote add origin git@github.com/username/project-repo.git
+$ git push origin master
+$ git remote add juan git@github.com/juan/project-repo.git
+$ git merge juan/feature
+$ git push origin master
+```
 
```

Comment:
Very close. Technically `git init project-repo` creates a `project-repo` directory. So you would want to do that first before changing into the directory.